packages/kit/src/exports/public.d.ts

```diff
@@ -724,6 +724,16 @@ export type HandleFetch = (input: {
 	fetch: typeof fetch;
 }) => MaybePromise<Response>;
 
+/**
+ * The [`init`](https://svelte.dev/docs/kit/hooks#Shared-hooks-init) will be invoked once as soon as the server is executed.
```

Comment:
The exact behaviour will depend on the adapter (some init immediately, some need to wait for the first request to get stuff from request context), but this change is _technically_ correct (best kind) and is slightly more descriptive

```suggestion
 * The [`init`](https://svelte.dev/docs/kit/hooks#Shared-hooks-init) will be invoked before the server responds to its first request
```